sdk/src/configs/contracts.ts

```diff
@@ -0,0 +1,257 @@
+import { ARBITRUM, ARBITRUM_GOERLI, AVALANCHE, AVALANCHE_FUJI } from "./chains";
+import { Address, zeroAddress } from "viem";
+
+const CONTRACTS = {
+  [ARBITRUM_GOERLI]: {
```

Comment:
I think it's fine to remove ARBITRUM_GOERLI from all the configs, as we in fact don't support it

it might be confusing for users if they would go with it without any effect

Reply:
Should we remove this in UI too? If so, I think it's better to remove it from everywhere in single and separate PR

Reply:
I think it just confuses potential users that we support goerli, but we don't